Sources/PackageDescription/Target.swift

```diff
@@ -753,13 +739,13 @@ public final class Target {
     @available(_PackageDescription, introduced: 999.0)
     public static func executableTarget(
         name: String,
-        group: TargetGroup = .package,
         dependencies: [Dependency] = [],
         path: String? = nil,
         exclude: [String] = [],
         sources: [String]? = nil,
         resources: [Resource]? = nil,
         publicHeadersPath: String? = nil,
+        packageAccess: Bool = true,
```

Comment:
what does "true" means in this case? that it exposes it symbols to other modules? or that it "can see" other module symbols?

Reply:
It's both. If another target also has this param set to true, this target exposes its package symbols to that target and also can see package symbols from that target, depending on which target imports which.

Reply:
so do both modules (definition and consumer) need to be set to "true" for this feature to work?

Reply:
Yes. In the example below, A cannot access package symbols from B, but B can access package symbols from C. If A imports C, A still cannot access package symbols from C.  

target A w/ packageAccess = false: 
`import B`  
 
target B w/ packageAccess = true: 
`import C` 

target C w/ packageAccess = true:

Reply:
Most target types have a default of `true`, right? So I would think of it the other way around, you set this to false in order to opt-out.

Reply:
It is set to true by default in the public static func that introduces this param. The Target initializer itself here has the param set to false because it's called by earlier versions of static funcs that should not have access to package symbols.